### DISCUSSION TITLE
Can't set model as property if it hasn't been persisted yet

### DISCUSSION THREAD

**Discussion by biebthesecond** [livewire/livewire#6457] [7. Bugs (archived)]
### Livewire version

3.0.0-beta.7

### Which PHP version are you using?

PHP 8.1

### Steps To Reproduce

1. Create and register livewire component in routes (my use case is a model detail page)
2. Set a model property in livewire component
```
public SomeModel $someModel;

public function mount() {
        $this->someModel = SomeModel::find(1);
}
``` 
3. create a Filament delete action (Filament version 3.0.21)
```
public function deleteAction(): Action
{
        return DeleteAction::make()
            ->record($this->someModel )
            ->icon('heroicon-o-trash')
            ->successRedirectUrl(route('home'));
}
```

4. On the website, click and confirm the delete action

### Additional information

This is the error I'm getting
https://flareapp.io/share/Bm0Zq3B7

I'm putting up an issue here because in the filament discord, a help request was put up regarding this issue. Reading that I assume it's possibly an issue with livewire.  

Link to filament discord post \/
https://discord.com/channels/883083792112300104/1141632058121265163

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by riasvdv** [comment]
I have a failing test for that here: #6272
